cmd/porter/bundle.go

```diff
@@ -77,6 +79,35 @@ func buildBuildCommand(p *porter.Porter) *cobra.Command {
 	return cmd
 }
 
+func buildBundleListCommand(p *porter.Porter) *cobra.Command {
+	opts := struct {
+		rawFormat string
+		format    printer.Format
+	}{}
+
+	cmd := &cobra.Command{
+		Use:   "list",
+		Short: "list bundles",
```

Comment:
Let's change this to be a bit more clear what we are listing. How about "List installed bundles"?

I see that used managed bundles down below which I really like. So maybe you have better wording to suggest? "List managed bundles"? Up to you but just "List bundles" I think is too vague, especially since duffle lists bundles in a different way than porter.

Reply:
Updated....